desk/lib/dm.hoon

```diff
@@ -187,7 +187,7 @@
           author.u.reply
           now
       ==
-    :-  pac
+    :-  pac(dex (~(del by dex.pac) id))
     (put:on:replies:c replies u.tim %| tombstone)
```

Comment:
It's not obvious to me this is correct. We're putting a tombstone in the replies of the post, but also deleting its index entry? The reply continues existing though. So, why delete it?

It's surprising to me anyway that we touch the `dex.pac` for replies at all (in the `%add` case too). I thought that only was for top-level keys. But maybe my mental model of reply handling for chat is off.

Reply:
I was just going off your previous comment about missing the `=.  dex.pac...` but yeah looking at it now what I originally had was correct. We add reply IDs to the dex so that they can be looked up by ID later.